apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -159,18 +159,18 @@ export default function LandingPage({
         {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
         <EnrolledWorkshops />
         {plCoursesStarted?.length >= 1 && (
-          <section>
+          <section id={'self-paced-pl'}>
             <Heading2>{i18n.plLandingSelfPacedProgressHeading()}</Heading2>
             {RenderSelfPacedProgressTable()}
           </section>
         )}
         {deeperLearningCourseData?.length >= 1 && (
-          <div>
+          <section>
```

Comment:
Updated this to add spacing between the Self-Paced PL section and the Online PL Courses section:

| Before | After |
| ---- | ---- |
| <img width="1015" alt="Before" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/e014b459-ba9d-44ae-a200-298270dd852f"> | <img width="1015" alt="After" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/a6f51b82-eb41-4fd9-bffb-43c18d312ab9"> |